rapids-cmake/cpm/thrust.cmake

```diff
@@ -90,18 +97,27 @@ function(rapids_cpm_thrust NAMESPACE namespaces_name)
   set(multi_value)
   cmake_parse_arguments(_RAPIDS "${options}" "${one_value}" "${multi_value}" ${ARGN})
 
+  set(post_find_code "if(NOT TARGET ${namespaces_name}::Thrust)"
+                     "  thrust_create_target(${namespaces_name}::Thrust FROM_OPTIONS)" "endif()")
+
   if(Thrust_SOURCE_DIR AND _RAPIDS_BUILD_EXPORT_SET)
     # Store where CMake can find the Thrust-config.cmake that comes part of Thrust source code
     include("${rapids-cmake-dir}/export/find_package_root.cmake")
+    include("${rapids-cmake-dir}/export/detail/post_find_package_code.cmake")
     rapids_export_find_package_root(BUILD Thrust "${Thrust_SOURCE_DIR}/cmake"
                                     ${_RAPIDS_BUILD_EXPORT_SET})
+    rapids_export_post_find_package_code(BUILD Thrust "${post_find_code}"
+                                         ${_RAPIDS_BUILD_EXPORT_SET})
   endif()
 
   if(Thrust_SOURCE_DIR AND _RAPIDS_INSTALL_EXPORT_SET AND to_install)
     include("${rapids-cmake-dir}/export/find_package_root.cmake")
+    include("${rapids-cmake-dir}/export/detail/post_find_package_code.cmake")
     rapids_export_find_package_root(INSTALL Thrust
                                     [=[${CMAKE_CURRENT_LIST_DIR}/../../rapids/cmake/thrust]=]
                                     ${_RAPIDS_INSTALL_EXPORT_SET})
+    rapids_export_post_find_package_code(INSTALL Thrust "${post_find_code}"
+                                         ${_RAPIDS_INSTALL_EXPORT_SET})
   endif()
```

Comment:
Maybe combine these two blocks? Setting a variable to `set(BUILD_TYPE BUILD|INSTALL)` and using `${RAPIDS_${BUILD_TYPE}_EXPORT_SET}` should make all but the paths for `find_package_root` the same.

Reply:
Done.